### DISCUSSION TITLE
Support of Deno runtime

### DISCUSSION THREAD

**Discussion by mnsrulz** [duckdb/duckdb-wasm#1543] [Ideas]
Why do you want this feature?
Now a days spinning up edge apis is easy with netlify, deno hosting, vercel and cloudflare workers. Most of them supports deno and wasm out of the box. Making this lib compatible to support would be beneficial to run duckdb instances at edge making it extremely performant. 

I've created a [poc](https://github.com/mnsrulz/duckdb-netlifyedge/blob/main/netlify/edge-functions/duckdb.ts) to integrate this but getting an error `Worker is not defined`

**Comment by carlopi** [comment]
Thanks, that would be amazing to have.

This looks for now like a netlify specific problem if you manage to deploy locally but not remotely, maybe someone else has experience they can share their solution.

**Reply by mnsrulz** [reply]
Thanks for the reply @carlopi. This particular problem is somewhat tied to the fact that deno deploy doesn't support web workers as of today. I came to know that from the issue reported in https://github.com/denoland/deploy_feedback/issues/256
So, I believe the question I am having right now is, whether it's possible to run the duckdb wasm without web worker support?
